### PR TITLE
Add editor-framework

### PR DESCRIPTION
**Pull request by jwu**
A framework based on Electron and Polymer, let's you easily create dockable panel software in Electron.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Nice!

Should go under `Tools`.

File: readme.md [hunk 1]
```diff
@@ -72,6 +72,9 @@ Some good apps written with Electron.
 - [cookies](https://github.com/hstove/electron-cookies) - Adds support for `document.cookie`.
 - [window](https://github.com/jprichardson/electron-window) - Easily create windows, pass arguments to renderer, and manage global window references.
 
+## Frameworks
+
+- [editor-framework](https://github.com/fireball-x/editor-framework) - Easily create dockable IDE software in Electron.
```
Comment (sindresorhus) [inline readme.md]: `in Electron` is implied, so it's moot.

How about:

> Easily create dockable IDE apps.

**Comment by sindresorhus** [comment]
And give the commit a proper title.

**Comment by jwu** [comment]
Nice suggestion, updated :)

**Pull request closed without merging**